src/main/java/frc/robot/commandgroups/BackAndShootCargoSort.java

```diff
@@ -15,8 +16,8 @@ public BackAndShootCargoSort(Shooter shooter,
                                  Conveyor conveyor,
                                  Flap flap,
                                  DoubleSupplier conveyorPower,
-                                 DoubleSupplier distanceFromTarget) {
+                                 DoubleSupplier distanceFromTarget, BooleanSupplier hasTarget, DoubleSupplier odomDistance) {
```

Comment:
You can remove `hasTarget` and `odomDistance` and just pass the value from the odometry if the robot doesn't see the target. (It's also applies to other commands, I just don't want to copy & paste)

Reply:
I see what you're saying. To make it like an optional double, or something like that. But, I like it less. Also, it somewhat ruins the idea of the playWithoutVision boolean, which if true, will make it so hasTarget will always return false.